### PR TITLE
Optional gradient

### PR DESCRIPTION
**Pull request by nitrag**
This allows the developer to disable the gradient view.

```
LightboxConfig.InfoLabel.gradientDisabled = true
```

### CODE REVIEW

File: Source/LightboxController.swift [hunk 1]
```diff
@@ -72,12 +72,13 @@ open class LightboxController: UIViewController {
 
   open fileprivate(set) lazy var overlayView: UIView = { [unowned self] in
     let view = UIView(frame: CGRect.zero)
-    let gradient = CAGradientLayer()
-    let colors = [UIColor(hex: "090909").alpha(0), UIColor(hex: "040404")]
-
-    view.addGradientLayer(colors)
-    view.alpha = 0
+    if(!LightboxConfig.InfoLabel.gradientDisabled){
```
Comment (vadymmarkov) [inline Source/LightboxController.swift]: Could you remove parentheses?

File: Source/Views/FooterView.swift [hunk 2]
```diff
@@ -43,7 +43,9 @@ open class FooterView: UIView {
     super.init(frame: CGRect.zero)
 
     backgroundColor = UIColor.clear
-    _ = addGradientLayer(gradientColors)
+    if(!LightboxConfig.InfoLabel.gradientDisabled){
```
Comment (vadymmarkov) [inline Source/Views/FooterView.swift]: Could you remove parentheses here as well?

File: Source/Views/FooterView.swift [hunk 3]
```diff
@@ -108,7 +110,11 @@ extension FooterView: InfoLabelDelegate {
 
   public func infoLabel(_ infoLabel: InfoLabel, didExpand expanded: Bool) {
     resetFrames()
-    _ = expanded ? removeGradientLayer() : addGradientLayer(gradientColors)
+    if(expanded) {
```
Comment (vadymmarkov) [inline Source/Views/FooterView.swift]: And here 😄

File: Source/Views/FooterView.swift [hunk 4]
```diff
@@ -108,7 +110,11 @@ extension FooterView: InfoLabelDelegate {
...
+        _ = removeGradientLayer()
+    }else if(!LightboxConfig.InfoLabel.gradientDisabled){
```
Comment (vadymmarkov) [inline Source/Views/FooterView.swift]: It's better to add a space before `else`.

**Comment by zenangst** [comment]
Hey @nitrag, mind fixing up the things that @vadymmarkov reported? :)

**Comment by nitrag** [comment]
Done.

**Comment by nitrag** [comment]
@vadymmarkov Changes have been made, not sure why Github not seeing them.

**Comment by onmyway133** [comment]
@nitrag Hi, sorry for late response. The design of Lightbox is to use `Hue` to display background gradient layer. If you want to disable it, you can just remove the `gradient layer` of `overlayView`. We try to not overcomplicate the project with many configurations. Hope you understand

**Pull request closed without merging**